pages/learn/more/collected-data.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Collected Data
+---
+# Data collected by balena
```

Comment:
Can you update these references to `balena` to `{{ $names.company.lower }}`? There's a templating system that will build the documentation fully.

Reply:
Replaced.